Makefile

```diff
@@ -162,3 +162,8 @@ register-pipelines: update ${BEAT_NAME}
 .PHONY: import-dashboards
 import-dashboards:
 	echo "APM loads dashboards via Kibana, not the APM Server"
+
+# Checks the changelogs for certains branches.
+.PHONY: check-changelogs
+check-changelogs:
```

Comment:
```suggestion
check-changelogs: ## @testing Checks the changelogs for certain branches.
```
 this would add help  when you run `make help`
```
# Example 1:
# unit: ## @testing Runs the unit tests without coverage reports.
#    name => unit
#    varname => None
#    category => testing
#    doc => Runs the unit tests without coverage reports.
```